sig-storage/charter.md

```diff
@@ -1,5 +1,30 @@
 # SIG Storage
 
+## Scope
+
+SIG Storage is responsible to deal with all the features and integrations related to storage.
+
+### In Scope
+
+This group mainly deal with:
```

Comment:
My thoughts on scope:  

We are mainly responsible for the code that bridges the gap between Kubernetes storage primitives (PersistentVolumeClaims, VolumeSnapshots, CSI drivers, etc) and Virtual Machine storage requirements (bootable/mountable VM disk images, disk expansion, hotplug support, etc).

APIs

VirtualMachineSnapshot
VirtualMachineRestore

Volume Support

ContainerDisk
PersistentVolumeClaim
DataVolume
Ephemral
EmptyDisk
HostDisk
ConfigMap
Secret

Compatibility

Hotplug Volume
virtio-fs
SCSI reservation

Tools

virtctl image-upload
virtctl guest-fs